spec/server_spec.rb

```diff
@@ -33,6 +33,12 @@ def get_nodes(accepts)
         httpcall.get('/nodes', 'Accept' => accepts)
       end
 
+      def get_version
+        uri = URI(@server.url)
+        httpcall = Net::HTTP.new(uri.host, uri.port)
+        httpcall.get('/version', 'Accept' => 'application/json')
```

Comment:
Looks like we're asking for `application/json`, but the server is going to send `text/plain`, yes? According to the RFC I guess this is okay:

> Note: HTTP/1.1 servers are allowed to return responses which are not acceptable according to the accept headers sent in the request. In some cases, this may even be preferable to sending a 406 response. User agents are encouraged to inspect the headers of an incoming response to determine if it is acceptable.

Though I think web machine is generally pretty strict about this and would return a 406...